source/images/buyers-guide/icon-wifi.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 25.8 18.8" xml:space="preserve"><style>.st0{fill:#666}</style><path class="st0" d="M21 10.4c-.2 0-.5-.1-.6-.2-4.3-3.6-10.4-3.6-14.6-.1-.4.4-1.1.3-1.4-.1-.4-.4-.3-1 .1-1.4 5-4.1 12.2-4.1 17.1 0 .4.4.5 1 .1 1.4-.2.3-.4.4-.7.4z" id="Path"/><path class="st0" d="M24.9 6.4c-.2 0-.5-.1-.7-.2C17.8.6 8.1.6 1.7 6.2c-.4.4-1 .3-1.4-.1-.4-.4-.3-1 .1-1.4 7.2-6.3 18-6.3 25.1 0 .4.4.5 1 .1 1.4-.1.2-.4.3-.7.3z" id="Path_1_"/><path class="st0" d="M16.9 14.4c-.2 0-.4-.1-.6-.2-2-1.4-4.7-1.4-6.7 0-.5.3-1.1.2-1.4-.2-.2-.5-.1-1.1.3-1.4 2.7-1.9 6.3-1.9 9 0 .5.3.6.9.2 1.4-.2.2-.5.4-.8.4z" id="Path_2_"/><path class="st0" d="M13 18.8c-.6 0-1-.4-1-1s.4-1 1-1 1 .4 1 1-.4 1-1 1z" id="Path_3_"/></svg>
```

Comment:
This has <style>'ing in here and a hard coded ID – should these be adjusted?

Reply:
We let `svgo` take care of that.